app/controllers/lettings_logs_controller.rb

```diff
@@ -68,17 +68,27 @@ def edit
   end
 
   def destroy
-    if @log
-      if @log.delete
-        head :no_content
-      else
-        render json: { errors: @log.errors.messages }, status: :unprocessable_entity
-      end
+    render_not_found and return unless @log
+
+    authorize @log, policy_class: LogPolicy
+
+    if @log.delete
```

Comment:
Just a friendly note that I'd expect this to change to a state-change vs. permanent delete.